e2e/helpers.ts

```diff
@@ -1,10 +1,13 @@
 import { promises as fsPromises } from 'fs';
 
 import { mutationTestReportSchema } from '@stryker-mutator/api/report';
-import { expect } from 'chai';
+import chai from 'chai';
 import path from 'path';
 import { calculateMutationTestMetrics, MetricsResult, Metrics } from 'mutation-testing-metrics';
 import { execSync, ExecException } from 'child_process';
+import { jestSnapshotPlugin } from "mocha-chai-jest-snapshot";
```

Comment:
Why the `mocha-chai-jest-snapshot` plugin vs the `chai-jest-snapshot` plugin?

Reply:
Good catch, I don't know how I mixed them up, I've fixed this now.